processor.py

```diff
@@ -0,0 +1,146 @@
+from copy import deepcopy
+import variable
+
+path = 'phonebook.txt'
+
+def check_file():
+    with open (path, 'a', encoding='UTF-8'):
+        return
+
+def open_file():
+    phone_book = []
+    new_phone_book = []
+    count = 0
+    with open (path, 'r', encoding='UTF-8') as file:
+        data = file.readlines()
+        for line in data:
```

Comment:
`file` уже является итератором
```suggestion
        for line in file:
```
Только в этом случает в каждом `line` не будет на конце `\n`